### PR TITLE
Difficulties

### PR DESCRIPTION
**Pull request by MegaApplePi**
## Status

Done

## Changelog

- add disambig notice in Beatmaps
- make difficulties a generic page
- remove sorting folders in Difficulites
- remove spreads and link to RC instead

## Related Issues

supersedes #359

### CODE REVIEW

File: wiki/Difficulties/en.md [hunk 1]
```diff
@@ -1,141 +1,59 @@
-Difficulty appropriate gameplay elements
-==========================================
+# Difficulties
 
-Star Rating and Web Icons
--------------------------
+*Not to be confused with: [Beatmaps](/wiki/Beatmaps).*
 
-![Star Rating Range](SR_range.png "Star Rating Range")
+*See also: [Ranking Criteria § Mapset](/wiki/Ranking_Criteria/#mapset).*
```
Comment (TPGPL) [inline wiki/Difficulties/en.md]: `§` into `->` or `>`?

File: wiki/Difficulties/en.md [hunk 2]
```diff
@@ -1,141 +1,59 @@
...
 
--   From the [Ranking Criteria](/wiki/Ranking_Criteria):
-    -   Below 1.5: Easy ![Easy icon](/wiki/shared/diff/easy-s.png "Easy icon")
-    -   Below 2.25: Normal ![Normal icon](/wiki/shared/diff/normal-s.png "Normal icon")
-    -   Below 3.75: Hard ![Hard icon](/wiki/shared/diff/hard-s.png "Hard icon")
-    -   Below 5.25: Insane ![Insane icon](/wiki/shared/diff/insane-s.png "Insane icon")
-    -   Above 5.25: Expert ![Expert icon](/wiki/shared/diff/expert-s.png "Expert icon")
+The difficulty of a beatmap *describes* the level of skill needed to complete a beatmap.
```
Comment (TPGPL) [inline wiki/Difficulties/en.md]: The difficulty of a beatmap *describes* the level of skill needed to complete it.

File: wiki/Difficulties/en.md [hunk 3]
```diff
@@ -1,141 +1,59 @@
...
 
-osu! difficulty spreads
---------------------------------------------------
+![Star rating ranges](SR_range.png "Star rating ranges")
 
-From the [Ranking Criteria](/wiki/Ranking_Criteria):
+From the [Ranking Criteria](/wiki/Ranking_Criteria), a star rating of:
 
-"**The difficulties in the mapset must be in a consecutive order. Easy or Normal can be skipped if the gap in the star rating spread allows it. If your mapset has two difficulties, one of them cannot be an Insane or Expert. The lowest difficulty must be below 2.0 stars.**"
+- 1.5 (or below) is Easy ![Easy icon](/wiki/shared/diff/easy-s.png "Easy icon")
```
Comment (TPGPL) [inline wiki/Difficulties/en.md]: 1.5★ (or below) is called Easy, maybe?
not sure about this one, same for the ones below?

File: wiki/Difficulties/en.md [hunk 4]
```diff
@@ -1,141 +1,59 @@
...
+- 2.25 (or below) is Normal ![Normal icon](/wiki/shared/diff/normal-s.png "Normal icon")
+- 3.75 (or below) is Hard ![Hard icon](/wiki/shared/diff/hard-s.png "Hard icon")
+- 5.25 (or below) is Insane ![Insane icon](/wiki/shared/diff/insane-s.png "Insane icon")
+- 5.25 (or below) is Expert ![Expert icon](/wiki/shared/diff/expert-s.png "Expert icon")
```
Comment (TPGPL) [inline wiki/Difficulties/en.md]: same star ratings for Expert and Insane?

File: wiki/Difficulties/en.md [hunk 5]
```diff
@@ -1,141 +1,59 @@
...
-    -   ![Easy icon](/wiki/shared/diff/easy-s.png "Easy icon")(1.4) ![Hard icon](/wiki/shared/diff/hard-s.png "Hard icon")(2.5) ![Hard icon](/wiki/shared/diff/hard-s.png "Hard icon")(3.6)
-    -   ![Easy icon](/wiki/shared/diff/easy-s.png "Easy icon")(1.4) ![Hard icon](/wiki/shared/diff/hard-s.png "Hard icon")(2.6) ![Insane icon](/wiki/shared/diff/insane-s.png "Insane icon")(3.8)
+### Web icons
 
--   It is generally allowed to skip the **[Easy]** difficulty if the **[Normal]** difficulty follows the general guidelines of an Easy map
-    -   ![Normal icon](/wiki/shared/diff/normal-s.png "Normal icon") ![Hard icon](/wiki/shared/diff/hard-s.png "Hard icon") ![Insane icon](/wiki/shared/diff/insane-s.png "Insane icon") ![Expert icon](/wiki/shared/diff/expert-s.png "Expert icon")
+The star rating ranges will determine which web icon is to be used on any given beatmap. Following the icon ranges would usually give you a good estimate to achieve a balanced difficulty spread. There are some guidelines (see [Ranking Criteria](/wiki/Ranking_Criteria/#mapset)) you can follow to improve the difficulty spread of your beatmapset.
```
Comment (TPGPL) [inline wiki/Difficulties/en.md]: will determine -> determines
is to be used -> is going to be used?

File: wiki/Difficulties/en.md (hunk 4 above)
Comment (MegaApplePi) [reply wiki/Difficulties/en.md]: the graphic and the star ratings don't match at all 😕

Comment (MegaApplePi) [reply wiki/Difficulties/en.md]: oh wait... never mind, I misinterpreted the chart

File: wiki/Difficulties/en.md (hunk 1 above)
Comment (MegaApplePi) [reply wiki/Difficulties/en.md]: idk, it's a Wikipedia thing... ¯\_(ツ)_/¯

**Pull request merged**